guide-ru.md

```diff
@@ -26,6 +24,8 @@
 
 - [În Română](guide-ro.md)
 
+- [По-русски](guide-ru.md)
```

Comment:
Russian translation is already this document. Removing it would make it consistent with other translations